### ISSUE TITLE
Linux Sublime Text 4 Invalid Certs

### ISSUE THREAD

**Issue by Astilex5** [Monokai/monokai-pro-sublime-text#149]
When Trying to download Monokai Pro from Package Control I get this error:

`startup, version: 4126 linux x64 channel: stable
executable: /opt/sublime_text/sublime_text
application: /opt/sublime_text
working dir: /home/christianlarson
packages path: /home/christianlarson/.config/sublime-text/Packages
state path: /home/christianlarson/.config/sublime-text/Local
zip path: /opt/sublime_text/Packages
zip path: /home/christianlarson/.config/sublime-text/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.141646
startup time: 0.225946
first paint time: 0.24914
reloading plugin Default.arithmetic
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.colors
reloading plugin Default.comment
reloading plugin Default.convert_color_scheme
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.html_print
reloading plugin Default.indentation
reloading plugin Default.install_package_control
reloading python 3.3 plugin 0_package_control_loader.00-package_control
reloading python 3.3 plugin Package Control.1_reloader
reloading python 3.3 plugin Package Control.2_bootstrap
reloading plugin Default.keymap
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.rename
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin ShellScript.ShellScript
reloading python 3.3 plugin Package Control.Package Control
plugins loaded
Package Control: Skipping automatic upgrade, last run at 2022-02-03 00:58:28, next run at 2022-02-03 01:58:28 or after
ignored packages updated to: ["Theme - Monokai Pro", "Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
Package Control: Error downloading package. HTTP exception InvalidCertificateException (Host packages.monokai.pro returned an invalid certificate ([SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:548))) downloading https://packages.monokai.pro/Theme%20-%20Monokai%20Pro/1.1.20/Theme%20-%20Monokai%20Pro.sublime-package.
error: Package Control

Unable to download Theme - Monokai Pro. Please view the console for more details.
ignored packages updated to: ["Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings`

Looks like the cert is all screwy.

**Comment by neupauer** [comment]
I'm not sure if it's related to this, but I'm also having trouble installing Monokai Pro on macOS (Sublime Text v4).

**Comment by Astilex5** [comment]
> I'm not sure if it's related to this, but I'm also having trouble installing Monokai Pro on macOS (Sublime Text v4).
> 
> **EDIT:**
> After investigation I was able to solve my problem as described here: https://github.com/Monokai/monokai-pro-sublime-text/issues/144#issuecomment-939157165

I'm sure that works but this should get fixed rather than worked around. There are certificates for a reason. Usually security reasons.

**Comment by Astilex5** [comment]
I attempted this on my M1 Mac Mini on macOS 12 just to see if I got a Different report.

This Line I am almost positive is the issue:

`Package Control: Error downloading package. HTTP exception InvalidCertificateException (Host packages.monokai.pro returned an invalid certificate ([SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:548))) downloading https://packages.monokai.pro/Theme%20-%20Monokai%20Pro/1.1.20/Theme%20-%20Monokai%20Pro.sublime-package.`

The Maintainer needs to update the SSL Certificate.

**Comment by Monokai** [comment]
Should be fixed now. It has to do with this issue: https://github.com/Monokai/monokai-pro-sublime-text/issues/144

TLDR: Sublime Text 4 / PackageControl doesn't support Let's Encrypt certificates. I reconfigured my server a bit, and had forgotten this. I now use Buypass certificates and all should work again.

**Comment by danajanssen** [comment]
It works, thanks! My editor feels "right" again :)

As a heads up for anyone else: as per [the install instructions](https://monokai.pro/sublime-text) I had to `Command Palette -> Monokai Pro: select theme` to get the colours correct. Only using `Preferences -> Select Theme` won't get the theme loaded correctly
